packages/system-tests/src/spectron/application.ts

```diff
@@ -120,15 +120,28 @@ export class SpectronApplication {
   private async checkWindowReady(): Promise<any> {
     await this.webclient.waitUntilWindowLoaded(60000);
 
-    // Pick the first workbench window here
+    // Pick the window that loads VSCode,
+    // added code to address Spectron limitation in Windows OS (https://github.com/electron/spectron/issues/60)
     const count = await this.webclient.getWindowCount();
-
-    for (let i = 0; i < count; i++) {
-      await this.webclient.windowByIndex(i);
-
-      if (/bootstrap\/index\.html/.test(await this.webclient.getUrl())) {
-        break;
+    if (count > 1) {
+      for (let i = 0; i < count; i++) {
+        await this.webclient.windowByIndex(i);
+        const title = await this.webclient.getTitle();
+
+        if (
+          process.platform === 'win32' &&
```

Comment:
NodeJs always returns `win32` regardless of which windows version is being used (https://nodejs.org/docs/latest-v8.x/api/process.html#process_process_platform).